src/main/java/edu/harvard/iq/dataverse/GlobalIdServiceBean.java

```diff
@@ -42,18 +36,27 @@ public interface GlobalIdServiceBean {
     
     Map<String,String> getMetadataForTargetURL(DvObject dvObject);
     
-    Map<String,String> lookupMetadataFromIdentifier(String protocol, String authority, String identifier);
-    
     DvObject generateIdentifier(DvObject dvObject);
     
     String getIdentifier(DvObject dvObject);
     
     boolean publicizeIdentifier(DvObject studyIn);
     
+    String generateDatasetIdentifier(Dataset dataset);
+    String generateDataFileIdentifier(DataFile datafile);
+    boolean isGlobalIdUnique(GlobalId globalId);
+    
+    public String getUrlPrefix();
+    public String getSeparator();
```

Comment:
No, but these can have default scope rather than public. FWIW: Apparently private methods were also added in Java 9, e.g. https://www.baeldung.com/java-interface-private-methods.